### ISSUE TITLE
Bump version to fix git vurnerability

### ISSUE THREAD

**Issue by raghu999** [telia-oss/github-pr-resource#193]
Github published security advisory which describes high severity bug in git client. A new resource version will install the latest version of git client [GHSA-qm7j-c969-7j4q ](https://github.com/git/git/security/advisories/GHSA-qm7j-c969-7j4q)

**Comment by itsdalmo** [comment]
Hey @raghu999 and thanks for the heads up! 

I've re-triggered the automated build on Docker hub  for the latest commit on `master`, and get the following `git` version on the current `dev` tag (`latest` is on `2.18.1`):
```
$ docker run teliaoss/github-pr-resource:dev /bin/sh -c "git --version"
git version 2.18.4
```

Seems like the fix was introduced in `2.18.3`, so based on the above a new patch release of `github-pr-resource` should trigger the automated builds and bring `latest` to a good state. Will do that now :+1:

**Comment by itsdalmo** [comment]
I've released [v0.19.2](https://github.com/telia-oss/github-pr-resource/releases/tag/v0.19.2) now and `latest` is currently running Git 2.18.4. For the record the credentials helper is not used by this resource, so we were not affected by the above vulnerability specifically, but keeping up with patch releases is a good idea anyway 🙌 

Closing this issue now since it has been resolved. Thanks again!
